app/index.html

```diff
@@ -173,13 +173,17 @@ <h5 class="modal-title">
                 <h5>References</h5>
                 <ul class="references">
                     <li>
-                        <a href="https://doi.org/10.1063/1.5090481">Atomic-Density representations for machine learning</a>
-                        <br/>MJ Willatt, F Musil, M Ceriotti - J. Chem. Phys, 2019
+                        <a href="https://doi.org/"10.21105/joss.02117>Chemiscope: interactive structure-property explorer for materials and molecule</a>
+                        <br/>G Fraux, RK Cersonsky, M Ceriotti - JOSS, 2020
                     </li>
                     <li>
                         <a href="https://doi.org/10.1039/c6cp00415f">Comparing molecules and solids across structural and alchemical space</a>
```

Comment:
@ceriottm do we want to keep these references here now that we have dataset specific reference lists?